File: py-codellama13B-FP-all/taskid-36_fizz_buzz-gen7.py
def fizz_buzz(n: int) -> int:
    """Return the number of times the digit 7 appears in integers less than n which are divisible by 11 or 13.
    >>> fizz_buzz(50)
    0
    >>> fizz_buzz(78)
    2
    >>> fizz_buzz(79)
    3
    """

    # 11, 13, 17, 19
    # 7, 11, 13, 17, 19
    # 11, 7, 13, 17, 19
    # 11, 13, 7, 17, 19
    # 11, 13, 17, 7, 19
    # 11, 13, 17, 19, 7

    # 11 7 13 17 19
    # 11 13 7 17 19
    # 11 13 17 7 19
    # 11 13 17 19 7
    count = 0
    for i in range(n):
        if i % 11 == 0 or i % 13 == 0:
            if '7' in str(i):
                count += 1
    return count

File: java-codellama7B-FP-all/taskid-36_fizz_buzz-gen7.java
import java.util.*;
import java.lang.reflect.*;
import org.javatuples.*;
import java.security.*;
import java.math.*;
import java.io.*;
import java.util.stream.*;
class Problem {
    // Return the number of times the digit 7 appears in integers less than n which are divisible by 11 or 13.
    // >>> fizzBuzz((50l))
    // (0l)
    // >>> fizzBuzz((78l))
    // (2l)
    // >>> fizzBuzz((79l))
    // (3l)
    public static long fizzBuzz(long n) {

        return LongStream.rangeClosed(3, n).filter(d -> ((d % 11) == 0) || ((d % 13) == 0)).mapToObj(Integer::new).map(i -> Integer.parseInt(i.toString())).filter(d -> ((d % 11) == 7) || ((d % 13) == 7)).count();
    }
    public static void main(String[] args) {
    assert(fizzBuzz((50l)) == (0l));
    assert(fizzBuzz((78l)) == (2l));
    assert(fizzBuzz((79l)) == (3l));
    assert(fizzBuzz((100l)) == (3l));
    assert(fizzBuzz((200l)) == (6l));
    assert(fizzBuzz((4000l)) == (192l));
    assert(fizzBuzz((10000l)) == (639l));
    assert(fizzBuzz((100000l)) == (8026l));
    }

}
